ginga/Control.py

```diff
@@ -897,6 +897,7 @@ def channel_image_updated(self, channel, image):
             self.logger.info("Large image update: %.4f sec" % (
                 split_time1 - start_time))
 
+            self.make_callback('active-image', channel.fitsimage)
```

Comment:
@pllim, this callback is not for when an image is set in a channel (the closest one for that is "image-set", available on the channel viewer).  This callback (perhaps unfortunately named) is for when the user changes the channel, even to one where there is an old image present.  Perhaps we should rename this callback.